src/Tasks/XslTransformation.cs

```diff
@@ -165,7 +165,11 @@ public override bool Execute()
                     {
                         using (XmlReader xr = xmlinput.CreateReader(i))
                         {
-                            xslct.Transform(xr, arguments, xmlWriter);
+                            if (UseTrustedSettings)
+                            {
+                                Log.LogMessageFromResources(MessageImportance.High, "XslTransform.SecuritySettingsViaUseTrustedSettings");
```

Comment:
This will get logged redundantly for each input - please move outside of the loop

Reply:
Nice catch. Thank you.  That's sad with ignoring the loop.